src/bank.rs

```diff
@@ -159,13 +159,14 @@ impl Bank {
             mint_account.tokens -= genesis_block.bootstrap_leader_tokens;
             bootstrap_leader_account.tokens += genesis_block.bootstrap_leader_tokens;
             self.accounts.store_slow(
+                true,
```

Comment:
It's unusual to see a hardcoded boolean passed to every instance of a function. Usually that's an indicator that there should be 2 (sometimes 3) separate functions. If the branch you'll always take is statically known, why pass a runtime parameter?  Also, no test for that new branch?

Reply:
@garious added test.  The bank will use this parameter once its checkpointing its state, so right now Accounts implements an expanded interface which is a superset of what the old bank needs and what is necessary for checkpointing.